internal/integration_test/spectest/threads/spec_test.go

```diff
@@ -26,6 +26,5 @@ func TestCompiler(t *testing.T) {
 }
 
 func TestInterpreter(t *testing.T) {
-	t.Skip() // TODO: Delete after implementing interpreter support
 	spectest.Run(t, testcases, context.Background(), wazero.NewRuntimeConfigInterpreter().WithCoreFeatures(enabledFeatures))
```

Comment:
Notice this is passing but I haven't added the threads feature flag yet, since that is public API. It means after this PR threads code technically is runnable without setting a flag.

Let me know if I should add the flag in this PR or a followup, it will touch a few files so I think a followup would be better if that temporary situation is ok

Reply:
let's do the flag addition after compiler is done!